hcipy/util/finite_difference.py

```diff
@@ -20,15 +20,19 @@ def generate_convolution_matrix(grid, kernel):
 		The matrix that applies the convolution.
 	'''
 	if hasattr(kernel, 'grid'):
-		if np.all( (abs(kernel.grid.delta - grid.delta) / grid.delta) < 1e-10):
+		if np.all((abs(kernel.grid.delta - grid.delta) / grid.delta) < 1e-10):
```

Comment:
Use `if not np.allclose(kernel.grid.delta, grid.delta)`. Its intentions are clearer and it's a little more advanced.

Reply:
Done